notebooks/working/investigating_deconfounder.py

```diff
@@ -0,0 +1,592 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.3.4
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
```

Comment:
Can you list out the hypotheses / cases that you are planning on investigating in this notebook to generate evidence and information about when and why the deconfounder can be expected to work well?

It'll help readers understand what they're looking at and looking for in the notebook.